bears/vcs/git/GitCommitBear.py

```diff
@@ -197,6 +207,36 @@ def check_body(self, body,
                                'HEAD commit. Please add one.')
             return
 
+        if host_site:
+            body_string = '\n'.join(body).lower()
+            keys = set()
+
+            if 'github' in host_site.lower():
+                keys = self.GITHUB_KEYS
+            elif 'gitlab' in host_site.lower():
+                keys = self.GITLAB_KEYS
+
+            if any(key in body_string for key in keys):
+                urls = re.findall(r'(https?://\S+)', body_string)
```

Comment:
Yeah. @jayvdb suggested use of full urls. But I'm working on making it configurable as well.